crates/api_models/src/payment_methods.rs

```diff
@@ -402,22 +427,6 @@ pub struct ListPaymentMethodResponse {
     pub payment_methods: Vec<ResponsePaymentMethodsEnabled>,
 }
 
-// impl ResponsePaymentMethodTypes {
-//     pub fn new(
-//         pm_type: RequestPaymentMethodTypes,
-//         connector: String,
-//         payment_method: api_enums::PaymentMethod,
-//     ) -> Self {
-//         Self {
-//             payment_method_type: pm_type.payment_method_type,
-//             payment_experience: pm_type.payment_experience,
-//             connector,
-//             card_networks: pm_type.card_networks,
-//             payment_method,
-//         }
-//     }
-// }
-
 #[derive(Eq, PartialEq, Hash, Debug, serde::Deserialize, ToSchema)]
 pub struct ListPaymentMethod {
```

Comment:
Consider renaming this and other types in this module to match the naming conventions.